src/scales/scale.time.js

```diff
@@ -408,8 +408,9 @@ function computeOffsets(table, ticks, min, max, options) {
 	return {start: start, end: end};
 }
 
-function ticksFromTimestamps(scale, values, majorUnit) {
+function ticksFromTimestamps(scale, values) {
```

Comment:
I'm not fan of this change because it makes the code harder to maintain by obfuscating the method inputs. Except for `_adapter`, I wouldn't assume any of the other scale computed values to be available at the time of calling this method.

Reply:
Not sure if we care about the size, but this way is 6 bytes smaller